python/tests/test_constraints.py

```diff
@@ -1139,9 +1139,9 @@ def test_constr_LBOUND(self):
 
             for t in range(self.T):
                 for bus in net.buses:
-                    self.assertEqual(u[bus.index_v_mag[t]],bus.v_max)
+                    self.assertEqual(u[bus.index_v_mag[t]],bus.v_max_reg)
                     self.assertEqual(u[bus.index_v_ang[t]],pf.BUS_INF_V_ANG)
-                    self.assertEqual(l[bus.index_v_mag[t]],bus.v_min)
+                    self.assertEqual(l[bus.index_v_mag[t]],bus.v_min_reg)
```

Comment:
This was my main concern when I knew we were going to have multiple voltage limits. By changing v_min and v_max here to reg limits we are no longer "leaving things as they currently were" since before these limits were not associated with voltage regulation. We should only use reg limits in constraints/functions/etc that are directly associated with voltage regulation. In particular, they should only be used in voltage magnitudes of buses that are actually regulated by transformers or shunt devices.